tests/common/include/tests/common/tools_propagator.inl

```diff
@@ -118,28 +117,33 @@ TEST_P(PropagatorWithRkStepper, propagator_rk_stepper) {
     constexpr std::size_t n_brl_layers{4};
     constexpr std::size_t n_edc_layers{7};
     vecmem::host_memory_resource host_mr;
-    const auto d = create_toy_geometry(host_mr, n_brl_layers, n_edc_layers);
+
+    // Construct the constant magnetic field.
+    using b_field_t = decltype(
```

Comment:
Now we just need to find a way to make the backend configurable from here, but let's do that when we switch to the solenoid